Games/types/Mafia/roles/cards/CatGiver.js

```diff
@@ -0,0 +1,24 @@
+const Card = require("../../Card");
+
+module.exports = class CatGiver extends Card {
+
+	constructor(role) {
+		super(role);
+
+		this.meetings = {
+			"Give Cat": {
+				states: ["Day"],
+				flags: ["voting"],
+				action: {
+					labels: ["giveItem", "cat"],
+					run: function () {
+						this.target.holdItem("Cat");
+						this.target.queueAlert("You have received a cat!");
+						this.target.data.catLady = this.actor;
```

Comment:
I think it's better to store catlady as a property of a cat, and not a property of the target player

this way, dropping the cat will remove references to the catlady